css/styles.css

```
@@ -140,6 +148,7 @@
   /* Font Families */
   --font-body: "Geist", -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, sans-serif;
   --font-heading: "Instrument Serif", Georgia, "Times New Roman", serif;
+  --font-mono: "Geist Mono", ui-monospace, "SF Mono", "Cascadia Code", Menlo, monospace;
```

Stylelint (17.9.0)

[error] Expected "Menlo" to be "menlo" (value-keyword-case)

(value-keyword-case)